src/components/forms/FieldWrapper.tsx

```diff
@@ -112,6 +119,8 @@ function FieldWrapper(props, ref) {
     requiredMark = true,
     tooltip,
     isHidden,
+    labelPrefix,
```

Comment:
@MrFlashAccount I wouldn't add `labelPrefix` 'cause it violates Design principles. I can't imagine any case for that.

Reply:
What design principles does it violate though?

Reply:
The labels and inputs should go with the pattern. We can't move labels and inputs inside the grid. It would complicate the navigation.

Reply:
Good point, thanks!